js/functions.js

```diff
@@ -0,0 +1,50 @@
+//Функция для проверки длины строки.
+function checkLengthString(string, maxValue) {
+  return (string.length <= maxValue);
+}
+checkLengthString('Строка',8);
+
+//Функция для проверки, является ли строка палиндромом.
+function checkStringPolindrom(string){
+  const newString = string.split('').reverse().join('');
+  return (string.toLowerCase().replaceAll(' ','') === newString.toLowerCase().replaceAll(' ',''));
+}
+checkStringPolindrom('Топот');
```

Comment:
Можно немного птимизировать
```suggestion
function checkStringPolindrom(string){
  const newString = string.toLowerCase().replaceAll(" ", "");
  return newString.split("").reverse().join() === newString;
}
checkStringPolindrom('Топот');
```